test/manual/gfont.js

```diff
@@ -0,0 +1,182 @@
+/*eslint "no-unused-expressions": 0*/
+/*eslint max-len: [2, 100, 4]*/
+/* globals describe:false, it:false, xit: false */
+/* globals before:false, beforeEach: false, afterEach: false */
+
+var Module = require('../../src/transpiled.js');
+var addManualSymbols = require('../../src/symbols-manual.js').addManualSymbols;
+var addGeneratedSymbols =
+  require('../../src/symbols-generated.js').addGeneratedSymbols;
+var expect = require('chai').expect;
+var sinon = require('sinon');
+global.atob = require('atob');
+
+describe('GFont', function() {
+  var rocky;
+  before(function() {
+    rocky = {
+      module: Module()
+    };
+    addManualSymbols(rocky);
+    addGeneratedSymbols(rocky);
+  });
+
+  describe('fonts_get_system_font', function() {
+    var fonts_get_system_font;
+    before(function() {
+      fonts_get_system_font = rocky.fonts_get_system_font;
+    });
+
+    it('returns null for some cases', function() {
+      expect(fonts_get_system_font(undefined)).to.be.null;
+      expect(fonts_get_system_font(null)).to.be.null;
+    });
+
+    xit('returns callback font for unknown keys', function() {
```

Comment:
That's Mocha's way of disable a test (`xit()`) or a whole set of tests (`xdescribe()`).